.github/workflows/push-deploy-website.yml

```
@@ -36,13 +36,9 @@
       url: https://docs-api.talawa.io
     steps:
       - uses: actions/checkout@v4
-      - uses: pnpm/action-setup@v4
-      - uses: actions/cache@v3
+      - uses: actions/setup-node@v4
         with:
-          path: docs/.docusaurus
-          key: docusaurus-${{ hashFiles('docs/**') }}
-          restore-keys: |
-            docusaurus-
+          node-version: '20.x' 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)